tasks/main.yml

```diff
@@ -1,3 +1,12 @@
 ---
- - import_tasks: install.yml
+- import_tasks: apt_install.yml
```

Comment:
Why is this always executed?

Reply:
because it installs apt packages or updates them

Reply:
It doesn't update them though

Reply:
now it does

Reply:
No, I meant it doesn't and shouldn't, this is done by our upgrade role.

Reply:
fixed